common/OpTestLpar.py

```diff
@@ -285,3 +285,86 @@ def lpar_code_update(self, i_image, imagecomponent):
             print l_msg
             raise OpTestError(l_msg)
 
+    ##
+    # @brief It will check existence of given linux command(l_cmd) on lpar
+    #
+    # @param l_cmd @type string: linux command
+    #
+    # @return BMC_CONST.FW_SUCCESS or raise OpTestError
+    #
+    def lpar_check_command(self, l_cmd):
```

Comment:
The coding guidelines weren't obvious but input parameters (like l_cmd) should actually be i_cmd.  l_ indicates a local variable and applies to things like l_msg below.  i_ indicates input parameter to function.